src/pages/buy/buy.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useChangeScreen } from '@common/hooks/use-change-screen';
+import { useCurrentAccount } from '@store/accounts/account.hooks';
+import { makeTransakUrl } from '@common/transak-helper';
+import { BuyLayout } from './buy.layout';
+import { okcoinUrl } from '@common/okcoin-helper';
+import { useActiveFiatProviders, useHasFiatProviders } from '@query/hiro-config/hiro-config.query';
+import { ScreenPaths } from '@common/types';
+
+export const BuyPage = () => {
```

Comment:
We should strive to have more components like this 👏🏼 

The next person that has to touch this file can ignore all styling-specific code (but drill down if necessary), and easily understand what data is used, and what behaviour it has 💯

Reply:
I've noticed there has been a move to using `Page` in the component name, is that something we should follow now for all `pages`? I find it to be inconsistent across the app. Sometimes it is even `...FullPage`.

Reply:
@beguene can you please respond to this?

Reply:
I am currently refactoring routes to components you are creating that are either being called `Page` or `FullPage` and I want to understand the difference ...or I am going to rename `FullPage` to just `Page` for consistency.

Reply:
I think we can keep 'Page', I have no real preference but we can do that from now on.

Reply:
Ok, great, thanks for feedback. 👍